clients/python/pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[tool.mypy]
+# Global options
+check_untyped_defs = true
+disallow_any_generics = false
+# disallow_untyped_defs: if True, it enforces things like `def __init__(self) -> CLASSNAME` or `def test_() -> None` which does not worth the effort
+disallow_untyped_defs = false
+follow_imports = "silent"
+# ignore_missing_imports: removes all the missing imports stuff from external libraries which is annoying to the least
+ignore_missing_imports = true
+namespace_packages = true
+no_implicit_reexport = true
+plugins = ["pydantic.mypy", "sqlalchemy.ext.mypy.plugin"]
+python_version = "3.10"
+show_column_numbers = true
+show_error_context = false
+strict_optional = false
+warn_redundant_casts = true
+warn_return_any = true
+warn_unused_configs = true
+warn_unused_ignores = true
+
+[tool.pydantic-mypy]
+# SEE https://docs.pydantic.dev/mypy_plugin/#plugin-settings
+init_forbid_extra = true
+init_typed = true
+warn_required_dynamic_aliases = true
+warn_untyped_fields = true
+
+
+[tool.pytest.ini_options]
+# SEE https://docs.pytest.org/en/7.1.x/reference/customize.html#pyproject-toml
+testpaths = [
+    "test",
+]
+# SEE https://pytest-asyncio.readthedocs.io/en/stable/concepts.html#test-discovery-modes
+asyncio_mode = "auto"
```

Comment:
Ah you have it here. So I guess you can remove it from your pytest calls?

Reply:
yes